src/cast_core/src/RollingCollections.cpp

```diff
@@ -269,50 +295,125 @@ struct OplogTailer : public RunOperation {
         }
     }
 
+    bool isRollingOplogEntry(const bsoncxx::document::view &doc, long *rollingMillis)
```

Comment:
c++isms would be to have this either

1. return `std::optional<long>` - return `make_optional(timesplit[2])` if found else `return std::nullopt` or
2. pass `rollingMillis` by non-const ref and just use regular assignment

Reply:
Resolved using std::optional.  Nice feature!